source/reference/minio-mc/mc-version.rst

```diff
@@ -15,141 +15,35 @@ Description
 
 .. start-mc-version-desc
 
-The :mc:`mc version` command enables, suspends, and retrieves the :ref:`versioning <minio-bucket-versioning>` status for a MinIO bucket.
+The :mc:`mc version` commands enable, disable, and retrieve the :ref:`versioning <minio-bucket-versioning>` status for a MinIO bucket.
 
 .. end-mc-version-desc
 
-.. tab-set::
+:mc:`mc version` includes the following subcommands:
```

Comment:
Somewhere in here suggest linking to the `object-versioning.rst` as a "For more information on versioning in MinIO, see blah" or however you want to structure that.